### PR TITLE
Create CODEOWNERS

### PR DESCRIPTION
**Pull request by stefannibrasil**
As part of our work to easily identify the maintainers of our open source projects, this PR adds a [CODEOWNER](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) file.

What it does:

- Automatically include people as reviewers on pull requests
- Provide a clear reference of who should be answering questions and making decisions
- Make it easy to search for projects which need a new owner when somebody leaves

### CODE REVIEW

**Comment by cpytel** [comment]
Thanks, @stefannibrasil. Does this mean that I must be the one to approve all PRs? I wouldn't want that. I'm happy to be the lead maintainer of a project, but everyone at thoughtbot should have the ability to work on the project and make changes without me being a gatekeeper.

**Comment by stefannibrasil** [comment]
> Thanks, @stefannibrasil. Does this mean that I must be the one to approve all PRs? I wouldn't want that. I'm happy to be the lead maintainer of a project, but everyone at thoughtbot should have the ability to work on the project and make changes without me being a gatekeeper.

@cpytel You're right: this means you'd be the one to approve all PRs. I'm looking at the docs here and what we can do is add thoughtbot as an org after you. So the PRs would only be merged after being approved either by you or anyone in the org. It feels redundant to add you *and* thoughtbot but this way we know who the codeowner is. How does that sound?

**Comment by cpytel** [comment]
That sounds good to me. Thanks!

**Review by cpytel — APPROVED** [review]

**Pull request merged**
